src/h5web/visualizations/shared/TooltipMesh.tsx

```diff
@@ -46,39 +38,39 @@ function TooltipMesh(props: Props): ReactElement {
       const { zoom } = camera;
       const projectedPoint = camera.worldToLocal(evt.unprojectedPoint.clone());
 
-      const xCoord = Math.floor(xCoordScale(evt.unprojectedPoint.x));
-      const yCoord = Math.floor(yCoordScale(evt.unprojectedPoint.y));
+      const abscissa = abscissaScale.invert(evt.unprojectedPoint.x);
+      const ordinate = ordinateScale.invert(evt.unprojectedPoint.y);
 
       showTooltip({
         tooltipLeft: projectedPoint.x * zoom + width / 2,
         tooltipTop: -projectedPoint.y * zoom + height / 2,
-        tooltipData: [xCoord, yCoord],
+        tooltipData: [abscissa, ordinate],
```

Comment:
`tooltipData` is now dumb: it only provides the abscissa and ordinate of the mouse in axis coordinates.